dev/zapier.ts

```diff
@@ -0,0 +1,558 @@
+const formatFlag = {
```

Comment:
could you please type this variable 
```suggestion
const formatFlag: Fig.Option = {
```